content/html/concepts/tags/terms/img/img.md

```diff
@@ -11,7 +11,7 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Used to add an image to the document. It can also be used inside an anchor tag to create an image link.
+Used to add an image or a gif to the document. It can also be used inside an anchor tag to create an image link.
```

Comment:
This seems like sort of an awkward place to put this? The `<img>` tag supports a multitude of formats, including `.gif`. Honestly, this entire entry could use a polish. Here's what I suggest for each section.

This line:

```suggestion
The `<img>` tag displays an image on the web page.
```

The sentence "It can also be used inside an anchor tag to create an image link." should be incorporated into the "Example" section below. We can also link "anchor tag" to the relevant entry (https://www.codecademy.com/resources/docs/html/tags/a).

**Syntax**:
All the list-items should be turned into complete sentences with periods (`.`) at the end.
  * The one about the `src` attribute should include a short list of common supported formats:
```
- `src`: the URL or filename of the image that supports a litany of formats (GIF, JPEG, PNG, etc.).`
```

**Example**:
Can we include an image of what the rendered `logo.png` actually looks like? This can be done by (1) adding the image to the `media/` folder and (2) referencing here via markdown with "![alt-text-here](https://raw.githubusercontent.com/Codecademy/docs/main/media/logo.png)" (no quotes, of course).